manala.proftpd/tests/0200_configs.yml

```diff
@@ -0,0 +1,51 @@
+---
+
+- name: "{{ test }}"
+  hosts: debian
+  become: true
+  vars:
+    manala_proftpd_configs_exclusive: true
+    manala_proftpd_configs:
+      - file: proftpd.conf
+        config:
+          - ServerName:        Manala ProFTPD Server
+          - DefaultRoot:       "~"
```

Comment:
Is `~` the value for `null` in proftpd configuration ? In this case, it could be natively handled by implementing it right there https://github.com/manala/ansible-roles/blob/12c73e555ffe00482e57c127348e77260e562414/manala.proftpd/templates/configs/_macros.j2#L56